docs/source/settings.rst

```diff
@@ -31,6 +31,26 @@ Whether to enable Spidermon.
 SPIDERMON_EXPRESSIONS_MONITOR_CLASS
 -----------------------------------
 
+
+Default: ``spidermon.python.monitors.ExpressionMonitor``
+
+A fully qualified path for a Python Class that should inherit from 
```

Comment:
Instead of ‘Python Class that should inherit from’, you could just say ‘subclass of’.